docs/controls/Gantt.md

```diff
@@ -377,6 +378,9 @@ var input = {
             value: new Date(2018, 5, 1).toISOString()
         }
     ],
+    pan: {
```

Comment:
Since this is disabled by default. Lets not include it here.

Reply:
I will remove it